src/theme/components/newsletterSignup.ts

```diff
@@ -0,0 +1,69 @@
+const NewsLetterSignup = {
+  baseStyle: {
+    alignItems: "center",
+    borderWidth: { base: "0 1px 1px 1px", md: "1px 1px 1px 0" },
+    maxWidth: "1280px",
+    width: "100%",
+    "div#info": {
+      alignItems: "flex-start",
+      alignSelf: "stretch",
+      flex: "1 1 0",
+      bg: "ui.bg.default",
+    },
+    "div#color-box": {
+      alignSelf: "stretch",
+    },
+    "div#pitch": {
+      width: { base: "100%", md: "50%" }, // It's a two-column layout >md
+      padding: {
+        base: "var(--nypl-space-s) var(--nypl-space-l) var(--nypl-space-l) var(--nypl-space-l)", // @todo For some reason using "padding:" with more than one shorthand value, e.g. l, sm, xxl, doesn't work.
```

Comment:
It's an issue with general Chakra values for the full CSS property. We know that you have to use the entire CSS variable when adding more than one value and as you have for the `md` case, when it's a single value you can pass the shorthand directly.